pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pycombo"
-version = "0.1.00"
+version = "0.1.01"
```

Comment:
oh, we should bump it to 0.1.02, I already used 0.1.01 for some minor fix
(we probably need to add a changelog file to keep records as well)